portfolio/src/main/webapp/script.js

```diff
@@ -92,4 +92,20 @@ function showProjects() {
         let projectButton = document.getElementById("project-button");
         projectButton.style.display = "none";
     }
+}
+
+/**
+ * Acquire a comment from /data and display it.
+ */
+function getComment() {
```

Comment:
consider making this an async function. then you won't need to do a bunch of promise chaining
https://dev.to/shoupn/javascript-fetch-api-and-using-asyncawait-47mp

Reply:
updated, looks a lot better